patientsearch/config.py

```diff
@@ -11,6 +11,7 @@ def decode_json_config(potential_json_string):
 
 SERVER_NAME = os.getenv("SERVER_NAME")
 SECRET_KEY = os.getenv("SECRET_KEY")
+SESSION_TYPE = os.getenv("SESSION_TYPE", 'filesystem')
```

Comment:
@ivan-c I suspect you won't love the choice of `filesystem` here - it is simply the lowest bar for a functional session.  The `null` default causes the same problem named in PR description.  Additional options require additional libs, but something like `redis` probably makes more sense.

Reply:
No big deal, we can always add redis later if needed. What's another service :grin: